onnxmltools/convert/common/_apply_operation.py

```diff
@@ -338,6 +347,25 @@ def apply_sigmoid(scope, input_name, output_name, container, operator_name=None)
     _apply_unary_operation(scope, 'Sigmoid', input_name, output_name, container, operator_name)
 
 
+def apply_selu(scope, input_name, output_name, container, operator_name=None, alpha=None, gamma=None):
+    _apply_unary_operation(scope, 'Selu', input_name, output_name, container, operator_name, alpha=alpha, gamma=gamma)
+
+
 def apply_hard_sigmoid(scope, input_name, output_name, container, operator_name=None, alpha=None, beta=None):
     _apply_unary_operation(scope, 'HardSigmoid', input_name, output_name, container, operator_name,
                            alpha=alpha, beta=beta)
+
+
+def apply_identity(scope, input_name, output_name, container, operator_name=None):
+    name = _create_name_or_use_existing_one(scope, 'Identity', operator_name)
+    container.add_node('Identity', input_name, output_name, name=name)
+
+
+def apply_softmax(scope, input_name, output_name, container, operator_name=None, axis=1):
+    name = _create_name_or_use_existing_one(scope, 'Identity', operator_name)
+    container.add_node('Softmax', input_name, output_name, name=name, axis=axis)
+
+
+def apply_normalization(scope, input_name, output_name, container, operator_name=None, axis=1, p=2):
+    name = _create_name_or_use_existing_one(scope, 'Identity', operator_name)
```

Comment:
>Identity [](start = 52, length = 8)

Also this one. I'm still not comfortable with this _apply_operation.py, what is the benefit of apply_softmax?

Reply:
apply_* is a high-level interface to apply an operation onto some variables. It's ultimate goal is to provide a way to declare computational graphs like Keras and Tensorflow. That is, ultimately we can feed some variables into an apply_* function and then a new variable will be declared and returned. We are still half-way to the destination as you can see that appy_* requires users to specify output names and doesn't return output type, which is not idea. Let me summarize our status and my design goal; the only different is (1).

[Current]
1. Apply_* consumes input names and output names
2. Apply_* automatically create operator names if needed
3. Apply_* hides some details for specifying ONNX attrubutes so that supporting multiple onnx versions is easier.

[Ultimate]
1. Apply_* consumes input variables and produce output variables
2. Apply_* automatically create operator names if needed
3. Apply_* hides some details for specifying ONNX node attrubutes so that supporting multiple onnx versions is easier.